docs/reference/modules/cache.md

```diff
@@ -0,0 +1,67 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/cache`
+
+**Alias:** `apos.cache`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `@apostrophecms/cache` module is a general-purpose cache implementation for improved performance in all areas where results can be retained and reused temporarily. Any number of namespaces can be used to separatly maintain data. It is powered by a MongoDB collection.
+
+## Methods
+
+### `async set(namespace, key, value, lifetime)`
+The `set()` method is asynchronous and should be called with an await to insure that any data is cached prior to trying to retrieve it. The method takes four parameters. The first, `namespace`, takes a string and is used to prevent collisions between keys when caching unrelated data. The `key` parameter takes a string and is the identifier that can be used to retrieve the item to be stored. The `value` parameter takes any JSON-friendly value and does not need to be stringified. The optional `lifetime` parameter takes the number of seconds for the value to exist as an integer. If set to zero or unspecified then there won't be an expiration time. However, the cache should not be used for primary storage as it could be cleared at some point.
+
+#### Example
+
+```javascript
+…
+// Check the cache for the data we want
+let specificData = await self.apos.cache.get('apiData', 'specificData');
```

Comment:
Use the URL you're fetch-ing as the key argument, not a made-up string ("specificData"). That's what you would naturally do if you were caching api results.

Reply:
Changed the examples.